wsmaster/che-core-api-workspace/src/test/java/org/eclipse/che/api/workspace/server/devfile/validator/DevfileSchemaValidatorTest.java

```diff
@@ -84,13 +84,17 @@ public void shouldThrowExceptionOnValidationOfNonValidDevfile(
   public Object[][] invalidDevfiles() {
     return new Object[][] {
       // Devfile model testing
+      {
+        "devfile/devfile_missing_metadata.yaml",
+        "The object must have a property whose name is \"metadata\"."
+      },
       {
         "devfile/devfile_missing_name.yaml",
```

Comment:
I would say that it's a test `devflie_missing_name_and_unknown_metadata_field`, should not we separate them to two cases?

Reply:
There are 2 cases now:
1) missing metadata (the newly added one)
1) missing name in metadata (the reworked existing one)

What else is missing?

Reply:
3. unknown field in the metadata
```
apiVersion: 1.0.0
metadata:
  something: else
```

The simplest devfile for missing name case can look like the following, or can't?
```
apiVersion: 1.0.0
metadata:
```